src/components/learningModule/LearningModule.js

```diff
@@ -1,69 +1,78 @@
-import React from 'react';
-import SelectionBox from '../selectionBox/SelectionBox';
-import Button from '../button/Button';
+import React from "react";
+import SelectionBox from "../selectionBox/SelectionBox";
+import Button from "../button/Button";
+import ProgressBar from "@ramonak/react-progress-bar";
 
-import './Styles.scss';
+import "./Styles.scss";
 
-const LearningModule = ({setGameStatus}) => {
+const LearningModule = ({ setGameStatus }) => {
   const [currentQuestionId, setCurrentQuestionId] = React.useState(0);
   const [quizData, setQuizData] = React.useState({});
-  let currentQuestion = quizData.questionArr ? quizData.questionArr[currentQuestionId]: {};
-  React.useEffect(()=>{
+
+  let currentQuestion = quizData.questionArr
+    ? quizData.questionArr[currentQuestionId]
+    : {};
+
+  React.useEffect(() => {
     getQuizData();
-  },[]);
+  }, []);
 
-  const getQuizData=()=>{
+  const getQuizData = () => {
     fetch("http://localhost:8080/problems")
-      .then((res)=>{
+      .then((res) => {
         return res.json();
-      }).then((data)=>{
+      })
+      .then((data) => {
         setQuizData(data);
-      }).catch((err)=>{
+      })
+      .catch((err) => {
         console.log(err);
       });
-  }
+  };
 
-  const handleSubmit=()=> {
-    if(currentQuestionId < quizData.totalQuestions-1){
-      setCurrentQuestionId(currentQuestionId+1);
+  const handleSubmit = () => {
+    if (currentQuestionId < quizData.totalQuestions - 1) {
+      setCurrentQuestionId(currentQuestionId + 1);
     } else {
       setCurrentQuestionId(0);
-      setGameStatus({message: "Great Job! Play again.", loadIntro: true});
+      setGameStatus({ message: "Great Job! Play again.", loadIntro: true });
     }
-  }
+  };
   let possibleAnswers = [];
-  if(currentQuestion.possibleAnswers){
+  if (currentQuestion.possibleAnswers) {
     possibleAnswers = currentQuestion.possibleAnswers.map((answer, index) => {
-      return <SelectionBox id={index} key={index} answer={answer} />
-    })
+      return <SelectionBox id={index} key={index} answer={answer} />;
+    });
   }
 
-
   return (
     <div className="learningModule">
-      { currentQuestion.title &&
+      <div className="learningModule--progressBar">
+        <ProgressBar
+          completed={(currentQuestionId / quizData.totalQuestions) * 100}
```

Comment:
Leyts pulll this out and  give it a variable name and run the calculcation outside if the jsx.